pkg/clients/aws/client_test.go

```diff
@@ -11,12 +11,15 @@ const fakeAccountNum = "123456789101"
 
 func TestGetRancherLicense(t *testing.T) {
 	tests := []struct {
-		name              string // name of the test, to be displayed on failure
-		hasNonEmeaLicense bool   // if the account has a license for the non-EMEA product sku
-		hasEmeaLicense    bool   // if the account has a licensed for the EMEA product sku
-		includeProductSku bool   // if the return from aws should include or exclude a product sku
-		desiredLicense    string // which license our client should pick - emea, non-emea, or nothing
-		errDesired        bool   // if we wanted an error for this test case
+		name                  string // name of the test, to be displayed on failure
+		hasNonEmeaLicense     bool   // if the account has a license for the non-EMEA product sku
+		hasEmeaLicense        bool   // if the account has a licensed for the EMEA product sku
+		hasTestNonEmeaLicense bool   // if the account has a license for the test non-EMEA product sku
+		hasTestEmeaLicense    bool   // if the account has a license for the test EMEA product sku
+		usesTestIds           bool   // if the account is using a test product id
+		includeProductSku     bool   // if the return from aws should include or exclude a product sku
```

Comment:
Are you actually asserting anything about the ProductSKU string? I don't think the bool value affects the test results and assertions.

Reply:
This is a configuration argument. IIRC this value is optional (i.e. it is a pointer) on the struct, so AWS doesn't have to return it. That being said, we need to return a license with this value filled in so that later uses can use this value when doing things like Product checkout (IIRC). This allows me to configure if "fakeAWS" returned a license with a product sku, and if not, that we still filled it in. I assert on the sku in L140.

Reply:
What value?